### PR TITLE
DOC-5703 - updated Docker links, fixed topic link, rewriting, moved topic

### PR DESCRIPTION
**Pull request by paulmoloneyr3**

### CODE REVIEW

File: content/en/platform/corda/4.10/community/docker-image.md [hunk 1]
```diff
@@ -7,32 +7,30 @@ date: '2023-01-10'
...
-and its own isolated process tree, separate from the host.
+Use `docker run` to create a writable container layer over the Docker image, and then start it. The container process that runs is isolated: it has its own file system, its own networking, and its own isolated process tree, separate from the host.
 
-* [Docker run](https://docs.docker.com/engine/reference/commandline/run/) is explained in full detail in the documentation published by Docker, Inc.
-* For information on using docker for development purposes, visit the [creating a node](generating-a-node.md) page.
-* If you have already set up your node and want to learn how to use your code using Docker, go to the [running a node locally](running-a-node.md) page.
-* If you want to learn more about Docker deployments and how to deploy to a compatibility zone, go to the [Docker deployments](node-docker-deployments.md) page.
+* `docker run` is explained in full detail [in the documentation published by Docker, Inc](https://docs.docker.com/engine/reference/commandline/run/).
```
Comment (AnnaK-R3) [inline content/en/platform/corda/4.10/community/docker-image.md:32]:
```suggestion
* `docker run` is explained in full detail [in the documentation published by Docker, Inc.](https://docs.docker.com/engine/reference/commandline/run/)
```

File: content/en/platform/corda/4.9/community/docker-image.md [hunk 2]
```diff
@@ -7,32 +7,31 @@ date: '2020-04-07T12:00:00Z'
...
 
 ## Prerequisites
 
-* Install and enable `Docker` and `docker-compose`. Docker CE (Community Edition) is sufficient. Docker, Inc. publish installation instructions for all major operating systems:
+* Install and enable Docker CE (Community Edition) and, optionally, Docker Compose. Docker CE is sufficient. Docker, Inc. publish installation instructions for all major operating systems:
   * [Docker CE](https://www.docker.com/community-edition)
-  * [Docker Compose](https://docs.docker.com/compose/install/).
-* Ensure you have a valid [node.conf file](node-database-tables.md) and set of certificates.
+  * [Docker Compose](https://docs.docker.com/compose/install/)
+* Ensure you have a valid [node.conf file]({{< relref "corda-configuration-file.md" >}}) and set of certificates.
```
Comment (AnnaK-R3) [inline content/en/platform/corda/4.9/community/docker-image.md:27]:
```suggestion
* Ensure you have a valid [`node.conf` file]({{< relref "corda-configuration-file.md" >}}) and set of certificates.
```

File: content/en/platform/corda/4.9/community/docker-image.md [hunk 3]
```diff
@@ -7,32 +7,31 @@ date: '2020-04-07T12:00:00Z'
...
-and its own isolated process tree, separate from the host.
+Use `docker run` to create a writable container layer over the Docker image, and then start it. The container process that runs is isolated: it has its own file system, its own networking, and its own isolated process tree, separate from the host.
 
-* [Docker run](https://docs.docker.com/engine/reference/commandline/run/) is explained in full detail in the documentation published by Docker, Inc.
-* For information on using docker for development purposes, visit the [creating a node](generating-a-node.md) page.
-* If you have already set up your node and want to learn how to use your code using Docker, go to the [running a node locally](running-a-node.md) page.
-* If you want to learn more about Docker deployments and how to deploy to a compatibility zone, go to the [Docker deployments](node-docker-deployments.md) page.
+* `docker run` is explained in full detail [in the documentation published by Docker, Inc](https://docs.docker.com/engine/reference/commandline/run/).
```
Comment (AnnaK-R3) [inline content/en/platform/corda/4.9/community/docker-image.md:33]:
```suggestion
* `docker run` is explained in full detail [in the documentation published by Docker, Inc.](https://docs.docker.com/engine/reference/commandline/run/)
```

**Review by AnnaK-R3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/4.9/community/docker-image.md (hunk 2 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.9/community/docker-image.md:27]: See above.

File: content/en/platform/corda/4.10/community/docker-image.md (hunk 1 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.10/community/docker-image.md:32]: As it's a sentence I want to retain the full stop.

**Review by AnnaK-R3 — APPROVED** [review]

**Review by AnnaK-R3 — APPROVED** [review]

**Pull request merged**